scripts/cdk.sh

```diff
@@ -57,6 +57,7 @@ for file in "${dir}"/mongodb-atlas-*.json; do
 			rm -rf l1-resources/"${path}"/src/*.ts
 		fi
 
+    # NOTE: known_issue MODULE_NOT_FOUND error.
```

Comment:
nit: does this need proper indentation? Also: what does this mean? I imagine that if someone new would pick up this file and see this comment there wouldn't be much context around it..

Reply:
i think marco that this error, govardhan just wanted to leave a note, is an error that he was having in his local, he is not able to run the cdk.sh, we were not able to find the reason, i suspect is a version error. it was working for me, so i just generated the files for him, if you think is necessary we can delete the comment